test/fixtures/basic/web.yml.erb

```diff
@@ -40,11 +40,11 @@ spec:
     metadata:
       labels:
         name: web
-        app: trashbin
+        app: basic
     spec:
       containers:
       - name: app
-        image: shopify/kubernetes-fixtures-hello-node:v1
+        image: nginx:alpine
```

Comment:
All this really needs to do is start successfully and not exit. As with the run-once pod, other ideas welcome.